Makefile

```diff
@@ -33,4 +33,7 @@ test_unit:
 test_features:
 	@bundle exec maze-runner -c features/*.feature
 
+test_new_feature:
+	@bundle exec maze-runner -c features/fastlane_build_report.feature
+
```

Comment:
This seems unnecessary to me - would you typically want to run just one feature and not all of them?  (For development purposes you can always just run `bundle exec maze-runner features/fastlane_build_report.feature`).  I wouldn't normally expect to have a `make` target for just one feature - unless it's a genuinely special case.

Reply:
Ah, is it because the existing tests have fallen out of maintenance and don't work, so you're wanting to make a fresh start with them?

Reply:
I didn't know about the `bundle exec maze-runner` method when I added that, so it was just so I could run the new tests exclusively.